src/components/SearchBar.tsx

```diff
@@ -0,0 +1,69 @@
+import { View, StyleSheet, TextInput } from 'react-native';
+import React from 'react';
+
+type SearchBarProps = {
+  setSearchValue: (text: string) => void;
+  searchValue: string;
+  membersData: MembersDataProps[];
+  setMembersData: ([]) => void;
+};
+
+type MembersDataProps = {
+  github_display_name: string;
+};
+
+const SearchBar = ({
+  setSearchValue,
+  searchValue,
+  membersData,
+  setMembersData,
+}: SearchBarProps) => {
+  const searchFunction = (text: string) => {
+    setSearchValue(text);
+    const updatedData = text
+      ? membersData?.filter((member) =>
+          member?.github_display_name
```

Comment:
Which members data we are fetching RDS or GitHub Orgs? If we are fetching RDS data why we display GitHub Display name not RDS name ?

Reply:
```
        const { github_display_name,first_name,last_name,username, github_id } = member;
        const assignedTo =   username ?? github_display_name  ?? github_id ?? first_name + last_name;  

```
used this as first_name + last_name is also null in most cases.